### PR TITLE
Issue #1633 Possibility to disable SBA server side by property

### PR DESCRIPTION
**Pull request by hammadirshad**

### CODE REVIEW

File: spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/SpringBootAdminServerEnabledCondition.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ */
+public class SpringBootAdminServerEnabledCondition extends SpringBootCondition {
+
+	@Override
+	public ConditionOutcome getMatchOutcome(ConditionContext context, AnnotatedTypeMetadata annotatedTypeMetadata) {
+		AdminServerProperties serverProperties = getClientProperties(context);
+
+		if (!serverProperties.getServer().isEnabled()) {
+			return ConditionOutcome
+					.noMatch("Spring Boot Server is disabled, because 'spring.boot.admin.client.enabled' is false.");
```
Comment (SteKoe) [inline spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/SpringBootAdminServerEnabledCondition.java:38]: Shouldn't this be `spring.boot.admin.server.enabled` in the `noMatch` string?

File: spring-boot-admin-docs/src/main/asciidoc/server.adoc [hunk 2]
```diff
@@ -52,7 +56,7 @@ In addition when the reverse proxy terminates the https connection, it may be ne
 
 | spring.boot.admin.probed-endpoints
 | For Spring Boot 1.x client applications SBA probes for the specified endpoints using an OPTIONS request.
-  If the path differs from the id you can specify this as id:path (e.g. health:ping)..
+If the path differs from the id you can specify this as id:path (e.g. health:ping)..
```
Comment (SteKoe) [inline spring-boot-admin-docs/src/main/asciidoc/server.adoc:59]: This change is not related to the PR, right? Can you please revert to removing the spaces?

**Review by SteKoe — CHANGES_REQUESTED** [review]

File: spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/SpringBootAdminServerEnabledCondition.java (hunk 1 above)
Comment (hammadirshad) [reply spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/SpringBootAdminServerEnabledCondition.java]: fixed in https://github.com/codecentric/spring-boot-admin/pull/1672/commits/de02259cfffb08ce802ef83473aeba9fbba1c4b6

File: spring-boot-admin-docs/src/main/asciidoc/server.adoc (hunk 2 above)
Comment (hammadirshad) [reply spring-boot-admin-docs/src/main/asciidoc/server.adoc]: fixed in https://github.com/codecentric/spring-boot-admin/pull/1672/commits/60f3c408107f5af069ddf02245be59fb32970f62
